.changeset/odd-paws-reply.md

```diff
@@ -0,0 +1,6 @@
+---
+'@primer/react-brand': patch
```

Comment:
Should be a minor as you're removing a prop

```suggestion
'@primer/react-brand': minor
```

Reply:
Great catch, thanks!